src/flamenco/vm/syscall/fd_vm_syscall_cpi_common.c

```diff
@@ -520,6 +745,7 @@ VM_SYSCALL_CPI_ENTRYPOINT( void *  _vm,
     return FD_VM_ERR_INSTR_ERR;
   }
   
+  //FD_LOG_NOTICE(("LEAVING CPI"));
```

Comment:
nit: remove

Reply:
Removing a bunch of prints i forgot to remove